models/mlflow/docker/Dockerfile

```diff
@@ -13,20 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG FROM_IMAGE=gpuci/miniforge-cuda
-ARG CUDA_VER=11.5
+ARG FROM_IMAGE="rapidsai/mambaforge-cuda"
+ARG CUDA_VER=11.5.1
 ARG LINUX_VER=ubuntu20.04
 ARG PYTHON_VER=3.8
-FROM ${FROM_IMAGE}:${CUDA_VER}-runtime-${LINUX_VER} AS base
+FROM ${FROM_IMAGE}:cuda${CUDA_VER}-base-${LINUX_DISTRO}${LINUX_VER}-py${PYTHON_VER} AS base
```

Comment:
@pdmack Thinks this might not need to derive from cuda here. Potentially investigate this.